templates/footer.html

```diff
@@ -1,21 +1,23 @@
 <!--
 	Footer has the following values that needs to be replaced when injected:
 
-	- ARTICLE_TITLE
+	- ARTICLE_TITLE - Pretty title to use in social sharing, looks like Ana Sayfa
+	- ARTICLE_NAME - Wiki-name to use in links, looks like Ana_Sayfa
+	- ARTICLE_ID - ID of the version this mirror was taken from, number
+	- ARTICLE_DATE - Date that the article was mirrored
 -->
 <div style="clear:both; background-image:linear-gradient(180deg, #E8E8E8, white); border-top: dashed 2px #AAAAAA; padding: 0.5em 0.5em 2em 0.5em; margin-top: 1em; direction: ltr;">
   <p>
-    This page was generated and distributed by the <a class="external text" href="https://github.com/ipfs/distributed-wikipedia">Distributed Wikipedia project</a>. It's based on the article issued from <a class="external text" href="https://tr.wikipedia.org/wiki/Ana_Sayfa?oldid=16812269">Vikipedi</a> - version of the 3/17/2016. The text is available under the <a class="external text" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution/Share Alike</a> but additional terms may apply for the media files. The Distributed Wikipedia Project is not affiliated to Wikipedia in any way.
+	This page was generated and distributed by the <a class="external text" href="https://github.com/ipfs/distributed-wikipedia">Distributed Wikipedia project</a>. It's based on the article issued from <a class="external text" href="https://tr.wikipedia.org/wiki/{{ARTICLE_NAME}}?oldid={{ARTICLE_ID}}">Vikipedi</a> - version of the {{ARTICLE_DATE}}. The text is available under the <a class="external text" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution/Share Alike</a> but additional terms may apply for the media files. The Distributed Wikipedia Project is not affiliated to Wikipedia in any way.
   </p>
   <p>
     From Snapshot: /ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S<br/>
```

Comment:
I updated this section to use the IPNS hash, which can be generated into the template before building a snapshot. I assume that the IPNS hash will vary based on the language we're building a snapshot of.